controllers/front/GuestTrackingController.php

```diff
@@ -111,94 +94,61 @@ public function postProcess()
      */
     public function initContent()
     {
-        parent::initContent();
+        FrontController::initContent();
 
         /* Handle brute force attacks */
         if (count($this->errors)) {
             sleep(1);
         }
 
-        $this->context->smarty->assign(array(
-            'action' => $this->context->link->getPageLink('guest-tracking.php', true),
-            'errors' => $this->errors,
-        ));
-        $this->setTemplate(_PS_THEME_DIR_.'guest-tracking.tpl');
+        // $this->context->smarty->assign(array(
```

Comment:
Leaving a few traps for future developers? :) If commented code is useless go ahead and remove it!